src/Parsers/Access/ASTAuthenticationData.cpp

```diff
@@ -131,7 +131,7 @@ void ASTAuthenticationData::formatImpl(const FormatSettings & settings, FormatSt
 
     if (password && !settings.show_secrets)
     {
-        prefix = "";
+        prefix += " `[HIDDEN]'";
```

Comment:
Let's change the secret's HIDDEN placeholder to **exactly** match the existing one, see `src/Parsers/ASTFunctionWithKeyValueArguments.cpp`, line 36. This one has a backtick instead of a single quote `'` and an excessive space character prepended.

It's important to keep everything in the same style, e.g. so that it could be searchable.